linux.cc

```diff
@@ -504,7 +502,7 @@ OSV_LIBC_API long syscall(long number, ...)
     SYSCALL2(flock, int, int);
     SYSCALL4(pwrite64, int, const void *, size_t, off_t);
     SYSCALL1(fdatasync, int);
-    SYSCALL6(pselect6, int, fd_set *, fd_set *, fd_set *, const struct timespec *, void *);
+    SYSCALL6(pselect6, int, fd_set *, fd_set *, fd_set *, const struct timespec *, const sigset_t*);
```

Comment:
Are you sure about this? The pselect(2) manual page says that `The final argument of the pselect6() system call is not a sigset_t * pointer, but is instead a structure of the form:`.

You even had a comment about this, which you removed in this PR.

How did you test this new implementation?

Reply:
Yeah, I have messed it up. I implemented this part to support one of the apps I was testing the static executables. But clearly, my tests have missed this bug.

Reply:
Addressed in the latest version of the PR.